EDAM_dev.owl

```diff
@@ -53380,6 +53380,9 @@ ows re-sequencing of complete genomes of any given organism with high resolution
 
     <owl:Class rdf:about="http://edamontology.org/topic_0199">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3321"/>
```

Comment:
Awesome, you've found the right way to add "parent" concepts (super-classes) @Lukong123 👍🏽 🙌🏽 🥇